shell/platform/common/cpp/client_wrapper/include/flutter/method_channel.h

```diff
@@ -51,8 +51,15 @@ class MethodChannel {
     messenger_->Send(name_, message->data(), message->size());
```

Comment:
This implementation should be replaced with calling the new method with a null `reply` to avoid duplicated code.

Reply:
Done.